wiki/Help_centre/Upgrading_to_lazer/it.md

```diff
@@ -0,0 +1,284 @@
+| difficolta' aperta come riferimento | ![Yes][true] | ![No][false] |
+| Per-oggetto SV / volume | ![No][false] | ![Yes][true] |
+| rotazione dei pattern | ![Yes][true] | ![Partial][partial][^editor-precise-rotation] |
+| ridimensionamento dei pattern | ![No][false] | ![Yes][true] |
+| caricamento della beatmap | ![Yes][true] | ![No][false] |
+| Storyboard editor | ![Yes][true] | ![No][false] |
+| Compatibilità cross-client | ![Yes][true] | ![Partial][partial][^incompatibilities] |
+
+## Passare a lazer
+
+Quindi hai decido di dare una chance a Lazer? Bene!
+
+Puoi trovare il download [Qui](https://github.com/ppy/osu#running-osu). Prossimamente, sarai in grado di passare a lazer direttamente da stable (dall'impostazione `Release stream`) e' trovare il link di download nel sito di osu!.
+
+## FAQ
+
+### Migrazione
+
+#### Stable se ne andrà via? Sono forzato a stare su Lazer?
+
+Stable continuerà a essere mantenuto finche' gli utenti lo utilizzeranno. Perlomeno, sara' mantenuto per diversi anni.
+
+#### Potrò importare tutti i miei dati da stable a lazer?
+
+Attualmente, beatmap, skin, punteggi, replay e collezioni possono essere importati in lazer. Nota bene, **le impostazioni non sono ancora state importate**, quindi dovrai ri-configurarle da zero.
+
+#### Se importo le mie mappe su lazer, verra' usato il doppio dello spazio?
+
+Se stable e lazer si trovano nello stesso disco, gli [hard links](/wiki/Client/Release_stream/Lazer/File_storage#via-hard-links) verranno usati per evitare il consumo di spazio aggiunto.
+
+in altri casi, nell'importazione delle beatmap usera' il doppio dello spazio.
+
+#### se disinstallo lazer, cio' causera' problemi alla cartella dove stable e' installato?
+
+No.
+
+#### se disinstallo stable, il contenuto importato da stable dara' problemi su lazer?
+
+No.
+
+#### Se installo lazer, saro' in grado di ritornare su stable?
+
+Si, lazer verra' sempre installato a fianco a stable. A meno che tu non decida di eliminare uno dei due, Entrambi saranno accessibili.
+
+#### posso importare i miei dati da lazer a stable?
+
+No. Non verra' supportato.
+
+Detto questo, i singoli score e mappe possono essere esportati da lazer e manualmente importati su stable per il momento.
+
+### Gameplay and scoring
+
+#### Se faccio uno score su lazer, verrà visualizzato sul mio profilo?
+
+gli score verranno mostrati sotto la sezione "Partite Recenti" ma non in "Migliore Performance" per ora.
+
+#### Se imposto un punteggio su lazer, darà punti performance?
+
+I punteggi avranno gia' i punti performance calcolati (puoi vederlo nella sezione "Partite recenti" del tuo profilo), ma non contribuiranno ancora al valore totale.
+
+#### Lazer usa ScoreV2?
+
+Lazer attualmente utilizza un nuovo sistema di punteggio sperimentale che e' simile a ScoreV2 ma non e' identico. questo e' ancora sotto sviluppo e stiamo raccogliendo pareri dai player in base a quanto bene va in vari contesti (solo play, classifiche, tornei etc.)
```

Comment:
"quanto bene va" doesn't seem appropriate for a wiki context. i'd rephrase it in a more formal way "...stiamo ancora raccogliendo pareri e opinioni dai giocatori in base al contesto in cui viene usato (solo play, classifiche, tornei etc.).